doc/nodejs.rst

```diff
@@ -0,0 +1,57 @@
+Usage with Node.js
+==================
+
+Although remoteStorage.js was initially written for being used in a
+browser, we do support using it in a Node.js environment as well.
+
+Installation
+------------
+
+Client library
+^^^^^^^^^^^^^^
+
+remoteStorage.js is available from npm::
+
+   $ npm install remotestoragejs
```

Comment:
This is already documented in https://remotestoragejs.readthedocs.io/en/latest/getting-started/how-to-add.html